Examples/MAX32655/Hello_World/Makefile

```diff
@@ -406,3 +406,6 @@ clean:
 
 # The rule to clean out all the build products.
 distclean: clean libclean
+
+# Include rules for flashing
+include $(MAXIM_PATH)/Tools/Flash/flash.mk
```

Comment:
Let's include this at the bottom of gcc.mk so we don't have to regenerate the project Makefiles

Reply:
What about in the board Makefiles? This could be used with other compilers.